App/src/main/java/com/gluonhq/chat/service/WaveService.java

```diff
@@ -323,7 +329,35 @@ public void updateUnreadList(String channelId, boolean add) {
             ex.printStackTrace();
         }
     }
-    
+
+    @Override
+    public BooleanProperty newVersionAvailable() {
+        BooleanProperty versionAvailable = new SimpleBooleanProperty();
+        String OS = System.getProperty("os.name").toLowerCase();
+        if (OS.contains("win") || OS.contains("mac")) {
```

Comment:
yes! 

Also installing packages on linux may need sudo access. we can try to open it via the default applicaion using `xdg-open` but it might fail on older systems as well :)